### PR TITLE
[FIX](regresstest) fix regress test for export with scp files 

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes

Issue Number: close #xxx

here is no need to scp files in single be 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

File: regression-test/suites/export/test_array_export.groovy
Comment (zgxme) [inline regression-test/suites/export/test_array_export.groovy:1]: > `SELECT * FROM ${tableName} t ORDER BY k1 INTO OUTFILE "file://${outFile}/";`

When single-fe single-be environment, export to `outFilePath`, not `outFile` temp directory.

**Comment by amorynan** [comment]
run buildall

**Comment by amorynan** [comment]
run buildall

**Review by hello-stephen — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**